circle.yml

```diff
@@ -1,17 +1,48 @@
+# This config uses parallelism. See https://circleci.com/docs/1.0/environment-variables/#parallelism
 machine:
   environment:
-    # Use the same port as Selenium server.
-    DRIVER_PORT: 4444
-    # Use the same base URL as Selenium server.
-    DRIVER_BASE_URL: wd/hub
+    browser_name: $(case $CIRCLE_NODE_INDEX in 0) browser_name='chrome' ;; 1) browser_name='firefox' ;; 2) browser_name='internet explorer' ;; 3) browser_name='android' ;; esac; echo $browser_name)
+    SAUCE_LOG: $CIRCLE_ARTIFACTS/sauce_log.txt
+    WATAI_TESTS: tests
+    SAUCE_USERNAME: cbenz
+    # SAUCE_ACCESS_KEY is defined in the CircleCI web UI
+    PORT: 2030 # Legislation Explorer ExpressJS server port
+
+dependencies:
+  post:
+    - npm install saucelabs # allow sending Watai results to SauceLabs
```

Comment:
Aaaah I get it: the `config-ci` is actually moved to `.watai/config`, which has lower precedence than `config`. The `views` are thus ignored… but I'm afraid that the `browser` instructions could be too!